features/admin.applications.v1/components/forms/advanced-configurations-form.tsx

```diff
@@ -200,19 +200,14 @@ export const AdvancedConfigurationsForm: FunctionComponent<AdvancedConfiguration
      */
     const isClientAttestationMethodsUiDisabled: boolean = !isEnableClientAttestation || isClientAttestationUIDisabled;
 
+    const isTrustedAppConsentRequired: boolean = useSelector((state: AppState) => {
```

Comment:
```suggestion
    const isTrustedAppConsentRequired: boolean = useSelector(
        (state: AppState) => state?.config?.ui?.isTrustedAppConsentRequired);
```
Let's use optional chaining to avoid NPE and arrow function with direct return